packages/orbit-design-tokens/src/js/__tests__/deprecatedTokens.test.ts

```diff
@@ -0,0 +1,643 @@
+    paletteInkLightActive: theme.palette.ink.lightActive,
+    paletteInkNormal: theme.palette.ink.normal,
+    paletteInkNormalHover: theme.palette.ink.normalHover,
+    paletteInkNormalActive: theme.palette.ink.normalActive,
+    paletteOrangeLight: theme.palette.orange.light,
+    paletteOrangeLightHover: theme.palette.orange.lightHover,
+    paletteOrangeLightActive: theme.palette.orange.lightActive,
+    paletteOrangeNormal: theme.palette.orange.normal,
+    paletteOrangeNormalHover: theme.palette.orange.normalHover,
+    paletteOrangeNormalActive: theme.palette.orange.normalActive,
+    paletteOrangeDark: theme.palette.orange.dark,
+    paletteOrangeDarkHover: theme.palette.orange.darkHover,
+    paletteOrangeDarkActive: theme.palette.orange.darkActive,
+    paletteOrangeDarker: theme.palette.orange.darker,
+    paletteRedLight: theme.palette.red.light,
+    paletteRedLightHover: theme.palette.red.lightHover,
+    paletteRedLightActive: theme.palette.red.lightActive,
+    paletteRedNormal: theme.palette.red.normal,
+    paletteRedNormalHover: theme.palette.red.normalHover,
+    paletteRedNormalActive: theme.palette.red.normalActive,
+    paletteRedDark: theme.palette.red.dark,
+    paletteRedDarkHover: theme.palette.red.darkHover,
+    paletteRedDarkActive: theme.palette.red.darkActive,
+    paletteRedDarker: theme.palette.red.darker,
+    paletteGreenLight: theme.palette.green.light,
+    paletteGreenLightHover: theme.palette.green.lightHover,
+    paletteGreenLightActive: theme.palette.green.lightActive,
+    paletteGreenNormal: theme.palette.green.normal,
+    paletteGreenNormalHover: theme.palette.green.normalHover,
+    paletteGreenNormalActive: theme.palette.green.normalActive,
+    paletteGreenDark: theme.palette.green.dark,
+    paletteGreenDarkHover: theme.palette.green.darkHover,
+    paletteGreenDarkActive: theme.palette.green.darkActive,
+    paletteGreenDarker: theme.palette.green.darker,
+    paletteBlueLight: theme.palette.blue.light,
+    paletteBlueLightHover: theme.palette.blue.lightHover,
+    paletteBlueLightActive: theme.palette.blue.lightActive,
+    paletteBlueNormal: theme.palette.blue.normal,
+    paletteBlueNormalHover: theme.palette.blue.normalHover,
+    paletteBlueNormalActive: theme.palette.blue.normalActive,
+    paletteBlueDark: theme.palette.blue.dark,
+    paletteBlueDarkHover: theme.palette.blue.darkHover,
+    paletteBlueDarkActive: theme.palette.blue.darkActive,
+    paletteBlueDarker: theme.palette.blue.darker,
+    paletteSocialFacebook: theme.palette.social.facebook,
+    paletteSocialFacebookHover: theme.palette.social.facebookHover,
+    paletteSocialFacebookActive: theme.palette.social.facebookActive,
+  };
+};
+
+describe("All deprecated tokens", () => {
+  const oldTokensObject = oldTokens();
+  const newTokensObject = createTheme();
+  it.each(Object.keys(oldTokensObject))("%s should exist in the return of createTheme", key => {
+    expect(key in newTokensObject).toBe(true);
```

Comment:
Generally avoid using `in` because it checks inherited properties as well, which we don't want.
```suggestion
    expect(newTokensObject).toHaveProperty(key);
```